### DISCUSSION TITLE
Character stats

### DISCUSSION THREAD

**Discussion by Thurston00** [EveronLife/EveronLife#83] [Ideas]
I noticed that there wasn't any standalone idea/ticket/issue outlining character stats.

The basics of character stats should include hunger and thirst (with configurable drain rates, maybe in the gamemode or a .conf settings file).

If you can think of any other stats which need to be added, feel free to comment below :)

**Comment by borknbeans** [comment]
Potentially a stamina stat that would affect stamina drain speeds, limiting sprint times. Additionally, it could also affect weapon steadiness if that is something we can control. I noticed we already had a stamina HUD icon added.

**Reply by LuxPoseidon** [reply]
Stamina is in game by default. Easy thing that can be done is altering the number for current stamina via script. ssooooo lets say you drink an energy drink.

while (length of effect)
{
    set currentstamina to 1
}

something like that would work fine

**Reply by LuxPoseidon** [reply]
as for altering regen or anything thats something I havent touched yet and would need to look into.

**Comment by DjCtavia** [comment]
I mainly see Thirst and Hunger.
**Breathing under water** would be a thing, if we're able to develop a feature for diving.

Don't see other stat that would add more realism or fun to gameplay. We will mostly have feature like drugs that will add an effect on a player for a duration so nothing like a stat thing.
